cmd/submit.go

```diff
@@ -19,12 +19,14 @@ import (
 
 // submitCmd lets people upload a solution to the website.
 var submitCmd = &cobra.Command{
-	Use:     "submit FILE1 [FILE2 ...]",
+	Use: "submit [FILE1	 ...]",
 	Aliases: []string{"s"},
 	Short:   "Submit your solution to an exercise.",
 	Long: `Submit your solution to an Exercism exercise.
 
     Call the command with the list of files you want to submit.
+    If you omit the list of files, the CLI will attempt to submit
+    automatically the appropriate solution files for the exercise.
```

Comment:
```suggestion
    the appropriate solution files for the exercise.
```

It may not matter if it is automatically, programmatically, or some other way, so I think we can remove that word (and if we do it magically instead later, we will not feel like we have to change this information.)

Reply:
I agree with this suggestion.